<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShapeModel</class>
 <widget class="QDialog" name="DialogShapeModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>741</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSnapOne">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>单帧采图</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ViewControl</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButtonPicOne">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>加载图片</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ViewControl</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>250</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前模板（双击查看模版图片）:</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>281</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>选择批头及对应螺丝编号</string>
   </property>
   <widget class="QComboBox" name="combo_Type">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>批头编号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_ShangStd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>螺丝编号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>螺丝30</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateShapeModel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>创建模板</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveOne">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ViewControl</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButtonPicContinue">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>连续采图</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">ViewControl</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ViewControl"/>
 </buttongroups>
</ui>
